### PR TITLE
add option to ignore symlinks

### PR DESCRIPTION
**Pull request by pebbleKite**
This is an attempt to address #4

### CODE REVIEW

**Comment by pebbleKite** [comment]
#4

**Comment by svenstaro** [comment]
Hi, thanks a lot! I had a look and a think and I now think that this is going the right way. However, we might want to go all the way with this. You implemented a custom directory rendering function for the version without symlinks. How about you restructure this in such a way that we always use a custom rendering function (as this would also allow us to make other changes such as changing the styling).

This way, you could then easily add a condition in that rendering function in case symlinks should be excluded.

Makes sense?

**Comment by pebbleKite** [comment]
The last commit (8034d4b2604dd3f5dc97c95e7da8e302e0044068) should address the single renderer.

File: src/main.rs [hunk 1]
```diff
@@ -127,9 +133,15 @@ pub fn parse_args() -> MiniserveConfig {
                 .validator(is_valid_auth)
                 .help("Set authentication (username:password)")
                 .takes_value(true),
+        ).arg(
+            Arg::with_name("no-symlinks")
+                .short("P")
+                .long("no-symlinks")
+                .help("Do not follow symbolic symbolic links"),
```
Comment (svenstaro) [inline src/main.rs]: Double `symbolic`

Comment (pebbleKite) [reply src/main.rs]: Sorry about that. fc46689 should fix that.

**Comment by svenstaro** [comment]
Thanks for the clean-up, will take a look soon!

**Comment by svenstaro** [comment]
Looks neat, thanks a lot.

**Pull request merged**